pimcore/lib/Pimcore/Bundle/AdminBundle/Controller/Admin/DataObjectHelperController.php

```diff
@@ -901,10 +1296,10 @@ public function importUploadAction(Request $request)
         $data = file_get_contents($_FILES['Filedata']['tmp_name']);
         $data = Tool\Text::convertToUTF8($data);
 
-        $importFile = PIMCORE_SYSTEM_TEMP_DIRECTORY . '/import_' . $request->get('id');
+        $importFile = PIMCORE_SYSTEM_TEMP_DIRECTORY . '/import_' . $request->get('importId');
```

Comment:
Add some kind of path travsering protection here? Looks kinda dangerous without checking in detail.